src/miscroutines.asm

```diff
@@ -145,15 +145,12 @@ BNE .end
 LDA $0DB9
 AND #$18
 if !THREEPLAYER || !!SMALLPLAYERS
-CMP #$08
 endif
 BEQ KillLuigi
 
 if !!PLAYERKNOCKBACK && !SMALLPLAYERS && !!THREEPLAYER
-LDA #$06
+LDA #$05
 STA $1504,x
-LDA #$2F
-STA $151C,x
```

Comment:
I know that that eliminates it ... but it is very uncomfortable while you are on a level with platforms that move in a straight line (room 00F) and jump to grab a mushroom, since the screen does not freeze, luigi is in the air and the platform leaves it behind letting it fall into a precipice, the animation would be much better if the screen freezes

Reply:
and when you're hit it's also the same case